### ISSUE TITLE
OSX Security error 

### ISSUE THREAD

**Issue by tovoss** [OpenShot/openshot-qt#3217]
**Describe the bug:**
After download and install of Openshot (Website & Latest daily build) OSX gives an security error when opening openshot the first time. 

**Steps to reproduce the behavior:**
1. Latest OSX
2. Install the Openshot DMG
3. Open Openshot

**Expected behavior:**
Program could be verified by OSX to allow opening the program.
**System Details:**
- OpenShot Version Latest daily build Feb 12 - 2020
- Operating System / Distro: 
System Software Overview:

  System Version:	macOS 10.15.3 (19D76)
  Kernel Version:	Darwin 19.3.0
  Boot Volume:	Macintosh HD
  Boot Mode:	Normal
  Computer Name:	TVO
  Username:	Tommy Voss (tommyvoss)
  Secure Virtual Memory:	Enabled
  System Integrity Protection:	Enabled
  Time since boot:	3 days 7:09

Hardware Overview:

  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro11,1
  Processor Name:	Dual-Core Intel Core i5
  Processor Speed:	2,6 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	3 MB
  Hyper-Threading Technology:	Enabled
  Memory:	16 GB
  Boot ROM Version:	158.0.0.0.0
  SMC Version (system):	2.16f68

**Log Files:**
No Log files exist as openshot never got opened

**Exception / Stacktrace:**
No Stack exist as openshot never got opened

**Screenshots:** (Optional)
<img width="425" alt="Screenshot 2020-02-12 at 16 56 12" src="https://user-images.githubusercontent.com/44980787/74353602-73c88b80-4dba-11ea-91cc-e19cdee2be17.png">
<img width="423" alt="Screenshot 2020-02-12 at 16 51 29" src="https://user-images.githubusercontent.com/44980787/74353608-762ae580-4dba-11ea-9ebd-7ad1ca5e9eae.png">

**Comment by SuslikV** [comment]
https://support.apple.com/en-us/HT202491
Apple says that this is security feature of the OS not an error.

**Comment by AdrienM76** [comment]
Just got this today after installing v2.5. (2.4.4 was working correctly) There is no way around this short of turning off Gatekeeper. (which I won't do)

The issue *may* be that the app needs to be notarized. (supposedly a requirement with Catalina, but that was initially relaxed) It *may* also need to be 'stapled'.

The following system check of the file via SecAssessment returns:
```spctl --assess --verbose /Applications/OpenShot*.app
/Applications/OpenShot Video Editor.app: accepted
source=Developer ID
```
As a comparison, a known notarized app returns a slightly different result:
```spctl --assess --verbose /Applications/Gnucash*.app 
/Applications/Gnucash.app: accepted
source=Notarized Developer ID
```

In between my last run of v2.4.4 and v2.5 I also applied the Catalina 10.15.3 security update. So the change might have been on Apple's end.

I managed to find this build:

OpenShot-v2.4.4-1554419061-d04a71ee-fbfac789-x86_64

and ran the check again, this time receiving:

```
/Applications/OpenShot Video Editor.app: a sealed resource is missing or invalid
```

Though this appears to be another problem, this build launches just fine.

Hope that helps.
